### PR TITLE
feat: implement just cmds `otel-stream`, `otel-stop`, and `otel-dev`

### PR DESCRIPTION
**Pull request by deniseli**
Part 1 of https://github.com/TBD54566975/ftl/issues/2020

This PR enables local testing of otel metric changes/additions. To use:

In tab 1, run `just otel-stream`.

In tab 2, run `just otel-dev`. You can pass in any args you'd run `ftl dev` with (e.g. `--recreate`). If you set `--log-level=DEBUG`, you should see `OTEL is enabled, exporting to localhost:4317` in your output. Any signals you trigger to log at this point should appear in the streaming output of tab 1.

When you're done, kill the `dev` command and then run `just otel-stop` in any tab other than tab 1.

### CODE REVIEW

**Comment by deniseli** [comment]
Completely forgot to integrate https://github.com/tobert/otel-cli before making the PR so I'll do that now :p

**Comment by deniseli** [comment]
Update: what we have here is already useful and needed for other folks to pick up tasks, so we're going to go ahead and merge it and I'll fix in the other improvements later.

To make this work locally, just open `internal/observability/client.go` and replace `otlpmetricgrpc.New(ctx)` with `otlpmetricgrpc.New(ctx, otlpmetricgrpc.WithInsecure(), otlpmetricgrpc.WithEndpoint(os.Getenv("OTEL_E\
XPORTER_OTLP_ENDPOINT")))`

File: Justfile [hunk 1]
```diff
@@ -173,4 +173,28 @@ debug *args:
...
+# Stop the docker container running otel.
+otelContainerID := `docker ps -f ancestor=otel/opentelemetry-collector:0.104.0 | tail -1 | cut -d " " -f1`
+otel-stop:
+  docker stop "{{otelContainerID}}"
+
+# Run `ftl dev` with the given args after setting the necessary envar.
+otel-dev *args:
+  #!/bin/bash
+
+  gprcPort=$(cat docker-compose.yml | grep "OTLP gRPC" | sed 's/:.*//' | sed -r 's/ +- //')
```
Comment (gak) [inline Justfile:198]: https://github.com/mikefarah/yq maybe easier in the future? (not an endorsement--I have not tried it!)

**Review by gak — APPROVED** [review]

File: Justfile (hunk 1 above)
Comment (gak) [reply Justfile:198]: Oh I see it's a comment, sorry that suggestion wouldn't work at all!

Comment (deniseli) [reply Justfile:198]: oooo nice! I'll go ahead and merge with the current stuff so folks can use it but that would be much nicer. I'll try to get it working with the real parser in the next update

Comment (deniseli) [reply Justfile:198]: Oh weird I just got your second comment on the comment gRPC string... yeah maybe it is actually safer to keep it like this. I'll think about it.

**Pull request merged**